src/charm.py

```diff
@@ -184,7 +193,7 @@ def apply_kube_ovn(self, service_cidr, registry):
             )
         else:
             # Only enable the mirror if a mirror interface is also provided
-            cni_args_to_replace["--enable-mirror"] = enable_global_mirror
+            cni_args_to_replace["--enable-mirror"] = str(enable_global_mirror).lower()
```

Comment:
😄 
True = "true" 
False =  "false"

I think this does the same thing
```suggestion
            cni_args_to_replace["--enable-mirror"] = json.dumps(enable_global_mirror)
```